### PR TITLE
correction : wrapping js into a function with api_key parameter to be called in html

### PR DESCRIPTION
**Pull request by ya33**

### CODE REVIEW

File: public/js/geocode.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
+function initializeGeocode(api_key) {
+  var map = L.mapotempo.map('map').setView([44.837778, -0.579197], 13);
+
+  var markers = [];
+  var markersGroup = L.layerGroup();
+  var geocode_url = `0.1/geocode.json?api_key=${api_key}`
```
Comment (fab-girard) [inline public/js/geocode.js]: ecma6

File: public/js/reverse.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+  var marker = undefined;
+
+  map.on('click', function(e) {
+    if (marker) {
+      map.removeLayer(marker);
+    }
+    marker = null;
+    $.ajax({
+      url: `0.1/reverse.json?api_key=${api_key}`,
```
Comment (fab-girard) [inline public/js/reverse.js]: ecma6

File: public/js/geocode.js (hunk 1 above)
Comment (ya33) [reply public/js/geocode.js]: corrected

File: public/js/reverse.js (hunk 2 above)
Comment (ya33) [reply public/js/reverse.js]: corrected

**Pull request merged**